sdk/python/kfp/v2/dsl/type_utils_test.py

```diff
@@ -21,18 +21,21 @@
 
 class TypeUtilsTest(unittest.TestCase):
 
-  _artifact_types = ['GCSPath', 'Model', 'Dataset', 'Schema', 'Metrics']
-  _non_artifact_types = ['String', 'Integer', 'Double', 'Arbitary', '', None]
+  _parameter_types = ['String', 'str', 'Integer', 'int', 'Float', 'Double']
```

Comment:
Sorry missed this part. Do you mind making this module-level private instead of static class attrs?

The rationale is to keep the structure as simple as possible: https://www.youtube.com/watch?v=aOEfIrC07XA&t=10m18s

Reply:
That's a good watch. Thanks :)